cats/cats-redis/src/main/java/com/netflix/spinnaker/cats/redis/cluster/ClusteredSortAgentScheduler.java

```diff
@@ -154,6 +154,28 @@ public void schedule(Agent agent, AgentExecution agentExecution, ExecutionInstru
   }
 
   @Override
+  public ClusteredSortAgentLock tryLock(Agent agent) {
+    // This score determines how much time the agent had left in the set it's currently in.
+    String releaseScore = agentScore(agent);
```

Comment:
Yes that's true - I wrote that comment because `releaseScore` can refer to either set. However, since it's only useful when `releaseScore` came from the WAITING set I'll fix the comment to make it clearer what the intention is.